.husky/pre-commit

```diff
@@ -1 +1,3 @@
+#!/bin/sh
+
```

Comment:
https://github.com/typicode/husky/releases/tag/v9.0.1#:~:text=shebang

Reply:
but got error on my windows, see the screenshoot

Reply:
git scm windows version still need shebang. and shebang still allowed by husky. here the doc

![image](https://github.com/user-attachments/assets/43e9a45b-b5e7-47b5-b0bd-b308c3595f95)

Reply:
I could see your screenshot of error. However, I could not reproduce on my window.